gui/wxpython/lmgr/frame.py

```diff
@@ -1137,7 +1137,7 @@ def GetMenuCmd(self, event):
             layer = self.GetLayerTree().layer_selected
             name = self.GetLayerTree().GetLayerInfo(layer, key="maplayer").name
             type = self.GetLayerTree().GetLayerInfo(layer, key="type")
-        except:
+        except (AttributeError, KeyError):
```

Comment:
If you use wrong key here, that's probably programmer's error, so I wouldn't use it here.